docker/Dockerfile.aarch64-unknown-linux-gnu.centos

```diff
@@ -0,0 +1,40 @@
+FROM ubuntu:20.04
+
+ARG DEBIAN_FRONTEND=noninteractive
+
+COPY lib.sh /
+COPY linux-image.sh /
+RUN /linux-image.sh aarch64
+
+FROM centos:7
+
+COPY common.sh lib.sh /
+RUN /common.sh
+
+COPY cmake.sh /
+RUN /cmake.sh
+
+COPY xargo.sh /
+RUN /xargo.sh
```

Comment:
a little side not, this is exactly the same as the x86_64 image, I believe we push all layers so we could technically share cache here.